test/react16_suggestors/react_dom_render_migrator_test.dart

```diff
@@ -458,6 +502,34 @@ main() {
       );
     });
 
+    test('no react_dom.dart import but usage has namespace', () {
+      testSuggestor(expectedPatchCount: 2, input: '''
+          part of 'a_file.dart';
+
+          main() {
+            react_dom.render(Foo()(), mountNode);
+          }
+        ''', expectedOutput: '''
+          part of 'a_file.dart';
+
+          main() {
+            react_dom.render(ErrorBoundary()(Foo()()), mountNode);
+          }
+        ''');
+    });
+
+    test('no react_dom.dart import but usage has namespace', () {
+      testSuggestor(expectedPatchCount: 2, input: '''
+          main() {
+            react_dom.render(Foo()(), mountNode);
+          }
+        ''', expectedOutput: '''
+          main() {
```

Comment:
Should an `over_react` import be added here since it's not in a part file and the import does not already exist?

Reply:
:okay: